Paymetheus/ViewModels/AutoBuyerViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) 2016 The Decred developers
+// Licensed under the ISC license.  See LICENSE file in the project root for full license information.
+
+using Paymetheus.Decred.Wallet;
+using Paymetheus.Framework;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    public class AutoBuyerViewModel : ViewModelBase
+    {
+        public AutoBuyerViewModel(AutoBuyerProperties properties) : base()
+        {
+            _autoBuyerProperties = properties;
+
+            StartAutoBuyerCommand = new DelegateCommand(StartAutoBuyerAction);
+            StopAutoBuyerCommand = new DelegateCommand(StopAutoBuyerAction);
+        }
+
+        public AutoBuyerViewModel AutoBuyer { get; }
+
+        private AutoBuyerProperties _autoBuyerProperties;
+
+        public AutoBuyerProperties AutoBuyerProperties
+        {
+            get { return _autoBuyerProperties; }
+            internal set { _autoBuyerProperties = value; RaisePropertyChanged(); }
+        }
+
+        public ICommand StartAutoBuyerCommand { get; }
+        public DelegateCommand StopAutoBuyerCommand { get; private set; }
+
+        private async void StartAutoBuyerAction()
```

Comment:
you don't want to use async void here, use async Task, and change DelegateCommand to DelegateCommandAsync